app/models/call.rb

```diff
@@ -27,6 +27,18 @@ class Call
                 track_destroy: true
   mongoid_userstamp user_model: 'User'
 
+  after_save :log_event, if: -> { status_changed? }
```

Comment:
We want to log an event on every call, so probably `after_create` and no need to make this conditional, right?